packages/wonder-blocks-button/components/button.stories.js

```diff
@@ -45,7 +45,7 @@ export const basicButtons = () => (
 );
 
 export const darkBackgroundButtons = () => (
-    <View>
+    <View style={{backgroundColor: Color.darkBlue}}>
```

Comment:
Just curious - why this change? Just happened to notice it looked better?

Reply:
The screenshots don't include the background color from the background addon so adding the background color here is necessary for it show up in the screenshot.  Also, making this change also shows off how the review workflow in the Chromatic app when there are changes to the images snapshots.